### ISSUE TITLE
Unsupported major.minor version 51.0 on Ubuntu 14.04

### ISSUE THREAD

**Issue by avilella** [dariober/ASCIIGenome#58]
I followed the installation instructions, but when trying to run on Ubuntu 14.04, I get:

```
Exception in thread "main" java.lang.UnsupportedClassVersionError: samTextViewer/Main : Unsupported major.minor version 51.0
	at java.lang.ClassLoader.defineClass1(Native Method)
	at java.lang.ClassLoader.defineClass(ClassLoader.java:648)
	at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:142)
	at java.net.URLClassLoader.defineClass(URLClassLoader.java:272)
	at java.net.URLClassLoader.access$000(URLClassLoader.java:68)
	at java.net.URLClassLoader$1.run(URLClassLoader.java:207)
	at java.net.URLClassLoader$1.run(URLClassLoader.java:201)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.net.URLClassLoader.findClass(URLClassLoader.java:200)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:325)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:270)
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:274)
	at org.eclipse.jdt.internal.jarinjarloader.JarRsrcLoader.main(JarRsrcLoader.java:56)
```

**Comment by dariober** [comment]
Hi- What version of java are you running? This error might be due to a version earlier than java 1.7. Try executing `java -version`, if you get 1.6 or earlier you should upgrade to the most recent (1.8).

**Comment by avilella** [comment]
Corrected it now by running with 1.8 instead of 1.6

**Comment by santhiya85** [comment]
Even after upgrading it to java8 , I get the following error

Exception in thread "main" java.lang.UnsupportedClassVersionError: com/mycompany/wordcounthadoop/WordCount : Unsupported major.minor version 52.0
	at java.lang.ClassLoader.defineClass1(Native Method)
	at java.lang.ClassLoader.defineClass(ClassLoader.java:803)
	at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:142)
	at java.net.URLClassLoader.defineClass(URLClassLoader.java:442)
	at java.net.URLClassLoader.access$100(URLClassLoader.java:64)
	at java.net.URLClassLoader$1.run(URLClassLoader.java:354)
	at java.net.URLClassLoader$1.run(URLClassLoader.java:348)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.net.URLClassLoader.findClass(URLClassLoader.java:347)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:425)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:312)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:358)
	at sun.launcher.LauncherHelper.checkAndLoadMain(LauncherHelper.java:482)
------------------------------------------------------------------------
BUILD FAILURE
